event/generate.go

```diff
@@ -161,7 +161,9 @@ func main() {
 
 	fmt.Fprint(file, `//go:generate go run generate.go
 
-// Documentation source: "Event reference" by Mozilla Contributors, https://developer.mozilla.org/en-US/docs/Web/Events, licensed under CC-BY-SA 2.5.
+// Package event defines markup to bind DOM events.
+//
+// See https://developer.mozilla.org/en-US/docs/Web/Events
```

Comment:
Nitpick. Given that this is a sentence, I'd consider not leaving out the period at the end.